build-thunder.make

```diff
@@ -3,4 +3,4 @@ core = 8.x
 includes[] = drupal-org-core.make
 projects[thunder][type] = profile
 projects[thunder][download][type] = git
-projects[thunder][download][tag] = 8.2.3
+projects[thunder][download][branch] = develop
```

Comment:
I will switch to the new tag, in a second PR, that just contains the release information.